node/src/main/scala/coop/rchain/node/configuration/model.scala

```diff
@@ -16,7 +16,9 @@ case class Server(
     dataDir: Path,
     mapSize: Long,
     inMemoryStore: Boolean,
-    maxNumOfConnections: Int
+    maxNumOfConnections: Int,
+    maxMessageSize: Int,
+    threadPoolSize: Int
```

Comment:
Shouldn't we mark the flags as not used/implemented until they are actually used?

Reply:
they are being used, see the files changed